docs/components.md

```diff
@@ -113,12 +132,8 @@ Ionic apps are made of high-level building blocks called Components, which allow
   <p>Radio inputs allow you to present a set of exclusive options.</p>
 </DocsCard>
 
-<DocsCard header="Refresher" href="api/refresher" icon="/icons/component-refresher-icon.png">
-  <p>Refresher provides pull-to-refresh functionality on a content component.</p>
-</DocsCard>
-
-<DocsCard header="Searchbar" href="api/searchbar" img="/icons/feature-component-search-icon.png">
-  <p>Searchbar is used to search or filter items, usually from a toolbar.</p>
+<DocsCard header="Refresher" href="api/refresher" img="/icons/feature-component-refresher-icon.png">
+  <p>A floating action button (FAB) is a circular button that offers an action on a screen.</p>
```

Comment:
This looks like the FAB description. I think the original Refresher description works well. `Refresher provides pull-to-refresh functionality on a content component.`